src/app/dialogs/edit-poll-dialog/edit-poll-dialog.component.ts

```diff
@@ -53,10 +53,8 @@ export class EditPollDialogComponent implements OnInit {
     this.loading = true;
 
     this.poll.name = this.validateForm.controls.name.value;
-    this.poll.startDate = this.validateForm.controls.date.value[0];
-    this.poll.endDate = this.validateForm.controls.date.value[1];
-
-    console.log(this.poll);
+    this.poll.startDate = this.validateForm.controls.date?.value ? this.validateForm.controls.date.value[0] : null;
+    this.poll.endDate = this.validateForm.controls.date?.value ? this.validateForm.controls.date.value[1] : null;
```

Comment:
replace with

```typescript
this.poll.startDate = this.validateForm.controls.date?.value;
this.poll.endDate = this.validateForm.controls.date?.value;
```

Instead of `null` this will evaluate to `undefined` if `date` is not present. The only usage I've found for `startDate` and `endDate` in this file is for an if check. The condition will work perfectly fine with `undefined`, e.g.

```typescript
if(this.poll.startDate) {...}
```

Reply:
you mean the following?
```
this.poll.startDate = this.validateForm.controls.date?.value[0];
this.poll.endDate = this.validateForm.controls.date?.value[1];
```

Reply:
yeah, sorry

Reply:
Should be fixed now